source/auth/tests/unified/reauthenticate_with_retry.json

```diff
@@ -1,6 +1,6 @@
 {
   "description": "reauthenticate_with_retry",
-  "schemaVersion": "1.12",
+  "schemaVersion": "1.3",
```

Comment:
The `auth/tests/unified` files are missing corresponding changes to YAML files.

Note that the YAML files are canonical, so the PHP script you have to check for minimum schema version should target those and then you should regenerate JSON files using the Makefile in `source/` for this repo.

Reply:
Good point. I don't know how it was missed by the script.